index.html

```diff
@@ -1,9 +1,77 @@
-Doina Fitchevici
-About
-Hello and welcome to my portfolio website! My name is Doina Fitchevici, and I am an aspiring front-end web developer. I am thrilled to embark on this journey to pursue my passion for web development and share my journey with you.
-Although I may currently lack professional experience in the field, I have dedicated significant time and effort to learning and honing my skills as a front-end web developer. I believe that with determination, perseverance, and a willingness to continuously learn and grow, I can make a positive impact in the world of web development.
-My interest in front-end web development sparked from a fascination with the power of the internet to connect people from all walks of life. The ability to create immersive and user-friendly websites that deliver engaging experiences to visitors is what truly excites me.
-Experience
-BeachMart.Inc, Harris Teeter, Junior Project, Asachi Lyceum. 
-Connect
-https://github.com/DoinaFitchevici
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Portfolio</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+    <main>
+    <section>
+    <h1>About</h1>
+    <p>
+    Hello and welcome to my portfolio website! My name is Doina Fitchevici, I am an aspiring front-end web developer, and also a mother of a two-year old boy. Besides enjoying being a mother, I am thrilled to embark on this journey to pursue my passion for web development and share my journey with you.
+    Although I may currently lack professional experience in the field, I have dedicated significant time and effort to learning and honing my skills as a front-end web developer.
+    </p>
+    </section>
```

Comment:
```suggestion
        <section>
            <h1>About</h1>
            <p>
                Hello and welcome to my portfolio website! My name is Doina Fitchevici, I am an aspiring front-end web developer, and also a mother of a two-year old boy. Besides enjoying being a mother, I am thrilled to embark on this journey to pursue my passion for web development and share my journey with you.
                Although I may currently lack professional experience in the field, I have dedicated significant time and effort to learning and honing my skills as a front-end web developer.
            </p>
        </section>
```
It will be easier to read and work with if indented here and below.